sources/packages/api/package.json

```diff
@@ -26,7 +26,8 @@
     "test:local": "NODE_ENV=local ENVIRONMENT=test jest",
     "test:local:win": "SET NODE_ENV=local&&SET ENVIRONMENT=test&&jest",
     "docker:start": "npm run setup:db && node dist/main",
-    "migration:create": "cd src/database/migrations && npx typeorm-ts-node-commonjs migration:create"
+    "migration:create": "cd src/database/migrations && npx typeorm-ts-node-commonjs migration:create",
+    "migration:revert": "npx typeorm-ts-node-commonjs migration:revert -d src/database/data-source.ts"
```

Comment:
just for my info, `cd src/database/migrations &&` not required for revert?

Reply:
Right, not required.